tensorflow_datasets/image/aflw2k3d_test.py

```diff
@@ -0,0 +1,34 @@
+# coding=utf-8
+# Copyright 2019 The TensorFlow Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for AFLW2000-3D dataset module."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from tensorflow_datasets import testing
+from tensorflow_datasets.image import aflw2k3d
+
+
+class Aflw2k3dTest(testing.DatasetBuilderTestCase):
+  DATASET_CLASS = aflw2k3d.Aflw2k3d
+  SPLITS = {
+      'test': 2,
+  }
+  DL_EXTRACT_RESULT = ''
```

Comment:
You might consider removing the DL_EXTRACT RESULT attribute if it is not used in the test file to clean up extra lines